global.json

```diff
@@ -9,7 +9,7 @@
   },
   "msbuild-sdks": {
     "Microsoft.DotNet.Arcade.Sdk": "9.0.0-beta.23559.3",
-    "Microsoft.DotNet.Helix.Sdk": "9.0.0-beta.23559.3",
+    "Microsoft.DotNet.Helix.Sdk": "9.0.0-beta.23563.1",
```

Comment:
This will be reverted, I bumped the version only to test the required fix for deep signing introduced in: https://github.com/dotnet/arcade/pull/14164